tests/api/test_result.py

```diff
@@ -16,53 +16,122 @@
 """
 import numpy as np
 import pytest
+from strawberryfields.backends.states import BaseGaussianState
 
 from strawberryfields.result import Result
 
 pytestmark = pytest.mark.api
 
 
+raw_results = {
+    "output": [np.ones((2, 3, 4, 5))],
+    "meta_array": np.array([1, 0, 1, 2]),
+    "meta_matrix": np.array([[1, 2], [3, 4]]),
```

Comment:
Might make sense to assign the values (`[np.ones((2, 3, 4, 5))]`, `np.array([1, 0, 1, 2])`, `np.array([[1, 2], [3, 4]])`) to variables above, to reduce code repetition when making assertions below.